lib/models/shared/model.skill.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the GNU General Public License v3
  * See LICENSE for details.
  */
-import RequestError from "lib/requestError";
+import RequestError from "../../../lib/requestError";
```

Comment:
This looks ugly, why are we re-defining `lib/` again?

Reply:
let's assume `./lib/` is our base path, `requestError` is in the root base dir (`./lib/`). So why are we trawling `$PWD/lib/requestError` instead of `../../requestError`? This module resolution path is wrong.

Reply:
This is because of `uvu` module resolution when launching tests. Need to dig more into their docs, but at first glance it didn't like the current path pattern in use. Running tests is fine with updated paths.

Reply:
Then it's not doing it's job at actually testing the API if it doesn't like the module resolution here.

Reply:
Got solved using proper `NODE_PATH` setting for `ts-node`